src/Protocol/Protocol_1_8.cpp

```diff
@@ -1147,6 +1147,25 @@ void cProtocol_1_8_0::SendResetTitle(void)
 
 
 
+void cProtocol_1_8_0::SendResourcePack(const AString & a_ResourcePackUrl)
+{
+	cPacketizer Pkt(*this, pktResourcePack);
+
+	cSha1Checksum Checksum;
+	Checksum.Update(reinterpret_cast<const Byte *>(a_ResourcePackUrl.c_str()), a_ResourcePackUrl.size());
```

Comment:
I'm pretty sure this is incorrect; it's supposed to be a checksum of the resource pack's contents (which vanilla has you specify in server.properties), not the URL.  I think the client uses it for caching, but I'm not 100% sure.

Reply:
Seems like you're right. At least it's not an urgent issue, but would be good for caching indeed.